README.md

```diff
@@ -173,3 +173,122 @@ Files is the API that lets us work with IPFS objects (DAG Nodes) as if they were
 #### Importer
 
 Importer are a set of layouts (e.g. UnixFS) and chunkers (e.g: fixed-size, rabin, etc) that convert data to a MerkleDAG representation inside IPFS.
+
```

Comment:
This segment, from here to Line 286, doesn't exist anymore, it is all on ROADMAP.md